user/pdfgrep/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "pdfgrep"
+pkgver = "2.2.0"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--prefix=/usr/"
+]
+hostmakedepends = [
+    "pkgconf",
+    "poppler-devel",
+    "libgcrypt-devel",
+    "pcre2-devel"
+]
+pkgdesc = "Tool to search text in PDF files"
+license = "GPL-2.0-or-later	"
```

Comment:
```suggestion
license = "GPL-2.0-or-later"
```
Remove superfluous whitespace